build.sbt

```diff
@@ -218,7 +218,6 @@ lazy val rdfProtos = (project in file("rdf-protos"))
         (ProtobufConfig / protobufRunProtoc).value,
       )
     }.dependsOn(crunchyProtocPlugin / generatePluginRunScript).value,
-    ProtobufConfig / protobufExcludeFilters := Seq(Glob(baseDirectory.value.toPath) / "**" / "grpc.proto"),
```

Comment:
The generated gRPC classes should not be in core, only in the grpc module. Please fix.